src/ngraph/runtime/plaidml/unit_test.manifest

```diff
@@ -39,265 +39,24 @@ topk_2d_min_one                         # No plans to implement TopK
-# unimplemented functionality, &c
-cos
+# unsupported op: `Erf`
 erf
-sin
-tan
-not
-abc_int64
-concat_matrix_int64
-select_double
-convert_int32_bool
-convert_float32_bool
-tensor_constant_int64
-constant_equality_bool
-numeric_float_inf
-numeric_double_inf
-computation_reuse
-pad_negative_exterior_1d_check_limits
-pad_edge_1d
-pad_edge_1d_top_neg
-pad_edge_1d_top_neg_bigger_than_tensor
-pad_edge_1d_bottom_neg
-pad_edge_1d_bottom_neg_bigger_than_tensor
-pad_edge_2d
-pad_edge_2d_with_neg
-pad_reflect_1d
-pad_reflect_1d_top_neg
-pad_reflect_1d_top_neg_bigger_than_tensor
-pad_reflect_1d_bottom_neg
-pad_reflect_1d_bottom_neg_bigger_than_tensor
-pad_reflect_1d_multi_reflect
-pad_reflect_2d
-pad_reflect_2d_with_neg
-pad_negative_exterior_2d
-pad_negative_exterior_2d_all_negative
-pad_negative_exterior_4d
-pad_symmetric
-max_trivial_int8
-max_trivial_5d_int32
-max_3d_to_scalar_double
-softmax_axis_3d
-logical_and
-logical_or
-batch_norm_inference_parameters_duplication
-batch_norm_fprop_b1c2h2w2
-batch_norm_fprop_b2c2h2w1
-batch_norm_fprop_b2c2d2h1w1
-batch_norm_fprop_inference_b2c2h2w1
-argmax_3D_axis_0
-argmax_3D_axis_1
-argmax_3D_axis_2
-argmin_trivial_in_double
-topk_2d_max_one_with_equal_values
-sum_trivial_in_double
-sum_stable_simple_double
-one_hot_vector_many_categories
-gather_no_axis_int8
-gather_no_axis_int16
-gather_no_axis_int32
-gather_no_axis_int64
-gather_no_axis_uint8
-gather_no_axis_uint16
-gather_no_axis_uint32
-gather_no_axis_uint64
-gather_no_axis_bool
-elu
-elu_negative_alpha
-prelu
-hardsigmoid
-prelu_shared_slope
-prelu_negative_slope
-relu_2Dfprop_i32
-conv_bias_1d
-conv_bias_2d
-conv_bias_3d
-conv_bias_bprop_2d
-conv_bias_add_2d
-space_to_depth
-depth_to_space
-normalize_across_chw_4d
-normalize_across_chw_4d_max_bias
-normalize_across_chw_3d
-normalize_across_chw_2d
-normalize_across_hw_4d
-gemm
-fused_clamp
-mvn_mean_normalization
-mvn_mean_normalization_split_channels
-mvn_mean_variance_normalization
-mvn_mean_variance_normalization_split_channels
-grn_4d
-grn_2d_with_bias
-scale_shift_no_broadcast
-scale_shift
-shuffle_channels_simple
-shuffle_channels_negative_axis
-shuffle_channels_float
-squeeze
-squeeze_default_axes
-squared_difference
-squared_difference_broadcast
-fake_quantize
-fake_quantize_with_clip
-fake_quantize_with_clip_across_channels
-dot_0_0
-dot_2x0_0
-equal
-notequal
-greater
-greater_int64
-greatereq
-less
-lesseq
-lesseq_int32
-lesseq_bool
-broadcast_vector_rowwise_int64
-minimum_int64
-maximum_int64
-auto_bcast_binary_elementwise
-any_trivial
-any_2x2x3_eliminate_dim_0
-backwards_acos
-backwards_asin
-backwards_atan
-backwards_softmax_all
-backwards_softmax_axis
-backwards_softmax_underflow
-backwards_softmax_3d
-batch_mat_mul_forward
-dot_matrix_2x0_0x2
-max_pool_uint8
-max_pool_int8
-avg_pool_uint8
-avg_pool_int8
-
-# Need erf
 gelu_f32
 gelu_f64
 gelu_backprop_factor_f32
 gelu_backprop_factor_f64
 backwards_gelu_f32
 backwards_gelu_f64
-
-# From onnx tests
```

Comment:
Ack. My `gtest_filter` didn't catch these ones when I was testing locally. I'll add them back in.